webservices/legal_docs/current_murs.py

```diff
@@ -19,76 +21,6 @@
     WHERE case_type = 'MUR'
 """
 
-MUR_SUBJECTS = """
-    SELECT subject.description AS subj, relatedsubject.description AS rel
-    FROM fecmur.case_subject
-    JOIN fecmur.subject USING (subject_id)
-    LEFT OUTER JOIN fecmur.relatedsubject USING (subject_id, relatedsubject_id)
-    WHERE case_id = %s
-"""
-
-MUR_ELECTION_CYCLES = """
-    SELECT election_cycle::INT
-    FROM fecmur.electioncycle
-    WHERE case_id = %s
-"""
-
-MUR_PARTICIPANTS = """
-    SELECT entity_id, name, role.description AS role
-    FROM fecmur.players
-    JOIN fecmur.role USING (role_id)
-    JOIN fecmur.entity USING (entity_id)
-    WHERE case_id = %s
-"""
-
-MUR_DOCUMENTS = """
-    SELECT document_id, category, description, ocrtext,
-        fileimage, length(fileimage) AS length,
-        doc_order_id, document_date
-    FROM fecmur.document
-    WHERE case_id = %s
-    ORDER BY doc_order_id, document_date desc, document_id DESC;
-"""
-# TODO: Check if document order matters
-
-MUR_VIOLATIONS = """
-    SELECT entity_id, stage, statutory_citation, regulatory_citation
-    FROM fecmur.violations
-    WHERE case_id = %s
-    ;
-"""
-
-OPEN_AND_CLOSE_DATES = """
-    SELECT min(event_date), max(event_date)
-    FROM fecmur.calendar
-    WHERE case_id = %s;
-"""
-
-DISPOSITION_DATA = """
-    SELECT fecmur.event.event_name,
-    fecmur.settlement.final_amount, fecmur.entity.name, violations.statutory_citation,
-    violations.regulatory_citation
-    from fecmur.calendar
-    inner join fecmur.event on fecmur.calendar.event_id = fecmur.event.event_id
-    inner join fecmur.entity on fecmur.entity.entity_id = fecmur.calendar.entity_id
-    left join (select * from fecmur.relatedobjects where relation_id=1) AS relatedobjects
-    on relatedobjects.detail_key = fecmur.calendar.entity_id
-    left join fecmur.settlement on fecmur.settlement.settlement_id = relatedobjects.master_key
-    left join (select * from fecmur.violations where stage='Closed' and case_id={0})
-    as violations on violations.entity_id = fecmur.calendar.entity_id
-    where fecmur.calendar.case_id={0} and event_name not in ('Complaint/Referral', 'Disposition')
-    ORDER BY fecmur.event.event_name ASC, fecmur.settlement.final_amount DESC NULLS LAST, event_date DESC;
-"""
-
-DISPOSITION_TEXT = """
-SELECT vote_date, action from fecmur.commission
-WHERE case_id = %s
-ORDER BY vote_date desc;
-"""
-
-STATUTE_REGEX = re.compile(r'(?<!\(|\d)(?P<section>\d+([a-z](-1)?)?)')
-REGULATION_REGEX = re.compile(r'(?<!\()(?P<part>\d+)(\.(?P<section>\d+))?')
-
 def load_current_murs():
```

Comment:
If most of the logic of this method is also relevant for ADRs, there is an opportunity to refactor some more.